server/conf/cms/secrets.sample.py

```diff
@@ -262,7 +280,6 @@
 
 - For each link used in the templating, add new links values (see example below).
 - New links must be added to the _PORTAL_AUTH_LINKS and _PORTAL_UNAUTH_LINKS lists.
-- The order of the _PORTAL_[…]_LINKS lists determine the rendering order of the elements.
```

Comment:
No, it does not. Never did. This is about an object whose documentation was originally from an array.